### PR TITLE
Book detail 3 - Structure

### PR DESCRIPTION
**Pull request by Julianhm9612**
## Summary
- Added new styles for the structure of book detail
- Added new file mixin
- Added new color variables

## Screenshot
Captura de Pantalla 2021-05-11 a la(s) 9 33 54 a  m

## Trello Card
https://trello.com/c/pkaxy4d9/28-detalle-de-libro-iii-estructura

### CODE REVIEW

File: books/scss/styles.scss [hunk 1]
```diff
@@ -1,5 +1,9 @@
 body {
+  background-color: $wild-sand;
+  display: flex;
   font-family: $roboto;
+  height: 100vh;
```
Comment (mcallegari10) [inline books/scss/styles.scss]: This won't be the case with every view. Are you sure you need this?

File: books/scss/mixin.scss [hunk 2]
```diff
@@ -0,0 +1,5 @@
+@mixin shadow($property) {
```
Comment (mcallegari10) [inline books/scss/mixin.scss:1]: This feels more like a SASS template than a mixin 🤔

**Review by mcallegari10 — CHANGES_REQUESTED** [review]

File: books/scss/styles.scss (hunk 1 above)
Comment (Julianhm9612) [reply books/scss/styles.scss]: It has been enhanced to cover getting more views. I use this to get the full height of the screen and don't burn sizes in px.

File: books/scss/mixin.scss (hunk 2 above)
Comment (Julianhm9612) [reply books/scss/mixin.scss:1]: I agree, his card does not need this mixin, but I will leave it for a future card.

**Review by mcallegari10 — APPROVED** [review]

**Pull request merged**